src/Widgets/DataFormUploadWidget/DataFormUploadWidgetComponent.tsx

```diff
@@ -50,14 +76,33 @@ provideComponent(DataFormUploadWidget, ({ widget }) => {
         </>
       ) : null}
 
-      <input
-        className="form-control"
-        type="file"
-        id={id}
-        multiple={widget.get('multiple')}
-        name={attributeName}
-        required={widget.get('required')}
-      />
+      <div {...getRootProps({ className: 'dropzone' })}>
+        <input
+          {...getInputProps({
+            id,
+            multiple: widget.get('multiple'),
+            name: attributeName,
+            required: widget.get('required'),
+          })}
+        />
+        {getDropMessage(widget.get('multiple'))}
+      </div>
+      <div>
+        <div className="d-flex flex-wrap mt-2 gap-2">
+          {attachments.map((attachment) => (
+            <Attachment attachment={attachment} key={attachment._id} />
+          ))}
+        </div>
+      </div>
     </div>
   )
 })
+
+function getDropMessage(multiple: boolean) {
+  switch (currentLanguage()) {
+    case 'de':
+      return `Wählen oder legen Sie ${multiple ? 'Dateien' : 'eine Datei'} hier ab.`
```

Comment:
* [ ] Haha: "Wählen [...] Sie eine Datei [...] ab.". How about:

Multiple Files:

	•	DE: Dateien auswählen oder hierher ziehen.
	•	EN: Choose files or drag them here.

Single File:

	•	DE: Datei auswählen oder hierher ziehen.
	•	EN: Choose a file or drag it here.